src/frontend/next/src/components/SearchInput/PostSearchInput.tsx

```diff
@@ -1,48 +1,50 @@
 import { ChangeEvent } from 'react';
-import { makeStyles } from '@material-ui/core/styles';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
 
 type PostSearchInputProps = {
   text: string;
   onChange: (e: ChangeEvent<HTMLInputElement>) => void;
 };
 
-const useStyles = makeStyles((theme) => ({
-  input: {
-    fontSize: '1.6rem',
-    '&:hover': {
-      border: '1px solid',
-      borderColor: theme.palette.primary.main,
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    input: {
+      fontSize: '1.6rem',
+      '&:hover': {
+        border: '1px solid',
+        borderColor: theme.palette.primary.main,
+      },
+      '&:focus': {
+        border: '2px solid',
+        borderColor: theme.palette.primary.main,
```

Comment:
Same here?

Reply:
Let's keep this for now. The release is tomorrow and then we will be moving to a new UI. So let's just make it functional for now.